go.mod

```diff
@@ -45,7 +45,18 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
-require golang.org/x/net v0.0.0-20210825183410-e898025ed96a
+require (
```

Comment:
Can you resolve those blocks by adding the dependencies to the existing ones + running `make modules`. I assume that's like with goimports, that only the blocks are formatted internally.